src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx

```diff
@@ -40,6 +40,14 @@ When you configure your log forwarder (our infrastructure monitoring agent, Flue
 
 Now your log files are enriched with enhanced metadata and contextual logging data. Your log forwarder sends the files to our logging endpoint for processing. From there you can use our log management capabilities to view, query, set up alerts, and more in New Relic.
 
+Here is an example of logs for your host's UI. You can see logs in context of events for the selected time period, and drill down into detailed data for any of the highlighted attributes. To take advantage of even more capabilities, click **Query logs** from here to go directly to the **Logs** UI.
```

Comment:
I like "here's" better but totally up to you!

Reply:
"It's" and "host's" sounded odd to me in same sentence. But yes, I'm still famous for not always using contractions.